ui/packages/shared/profile/src/ProfileMetricsGraph/index.tsx

```diff
@@ -30,6 +30,7 @@ interface ProfileMetricsGraphProps {
   profile: ProfileSelection | null;
   from: number;
   to: number;
+  filterByFunction: string | undefined;
```

Comment:
Instead of '| undefined' it should suffice to make this an optional prop like so:

```suggestion
  filterByFunction?: string;
```